src/hotspot/share/jfr/metadata/metadata.xml

```diff
@@ -848,6 +848,19 @@
     <Field type="float" contentType="hertz" name="switchRate" label="Switch Rate" description="Number of context switches per second" />
   </Event>
 
+  <Event name="CompilerQueueUtilization" category="Java Virtual Machine, Compiler" label="Compiler Queue Utilization" period="everyChunk">
+    <Field type="CompilerType" name="compiler" label="Compiler" />
+    <Field type="long" contentType="hertz" name="ingress" label="Ingress" description="Number of added requests per second"/>
+    <Field type="long" contentType="hertz" name="egress" label="Egress" description="Number of removed requests per second"/>
+    <Field type="long" name="size" label="Size"/>
+    <Field type="long" name="peak" label="Peak Size"/>
+    <Field type="long" name="added" label="Added"/>
+    <Field type="long" name="removed" label="Removed"/>
+    <Field type="long" name="totalAdded" label="Total Added"/>
+    <Field type="long" name="totalRemoved" label="Total Removed"/>
+    <Field type="int" name="compilerThreadCount" label="Number of Compiler Threads for this queue"/>
```

Comment:
The label should be short and use headline-style capitalization, how about "Compiler Thread Count"?
https://docs.oracle.com/en/java/javase/21/docs/api/jdk.jfr/jdk/jfr/Label.html

Reply:
Fixed, thank you for the reference